apps/journeys-admin/src/components/SingleJourney/CardOverview/CardOverview.spec.tsx

```diff
@@ -1,13 +1,30 @@
 import { render } from '@testing-library/react'
+import { renderWithApolloClient } from '../../../../test/testingLibrary'
+
 import CardOverview from './CardOverview'
+import { steps, oneStep } from './CardOverviewData'
 
 describe('CardOverview', () => {
-  it('should render button with the correct link', () => {
-    const { getByRole } = render(<CardOverview slug={'my-journey'} />)
-    expect(getByRole('link')).toBeInTheDocument()
-    expect(getByRole('link')).toHaveAttribute(
-      'href',
-      '/journeys/my-journey/edit'
+  it('should render description for no cards', () => {
+    const { getByText } = render(<CardOverview slug={'my-journey'} />)
+    expect(getByText('No cards in this journey')).toBeInTheDocument()
+  })
+  it('should have edit button', () => {
```

Comment:
```suggestion
  it('should have edit button when cards present', () => {
```
And also check the link to the edit sends us to the right place

In the future well also have a test `it('should have add button when cards not present',`